src/components/AnalysisPage/3dModelViewer/setupThreeScene.js

```diff
@@ -128,6 +138,16 @@ const setupThreeScene = (canvas) => {
     });
   };
 
+  const toggleMesh = (fileType) => {
```

Comment:
I think this function could use a bit more detailed of a name to include what its toggling between. Just so in the future if we have like 10 more functions to modify the 3d model scene this will be more clear what its doing